org_status/org_status.py

```diff
@@ -148,55 +174,78 @@ def main():
 
     all_repositories = []
 
-    for Host, org in generate_fetch_jobs(args.orgs):
-        token = None
+    if args.use_repo_list:
+        repos = decode_repo_list(args.use_repo_list, args.format, styled)
 
-        if not args.skip_host_checks:
-            try:
-                if not Host.get_host_status():
-                    print(styled(f'{Host.HostName} is currently down', 'red'))
-                    continue
-                else:
-                    hosts_only_print(f'{Host.HostName} is up')
-            except NotImplementedError:
-                verbose(
-                    f'{Host.HostName} does not support checking host status')
+        for Host, org in generate_fetch_jobs(args.orgs):
+            repo_names = []
+            repo_urls = []
+            for repo in repos:
+                repo_names.append(repo[0])
+                if repo[1].split('.com')[0].split('//')[1] == Host.HostName:
```

Comment:
Splitting by `.com` doesn't seem like the most reliable method. Perhaps look into using a url parsing library such as `furl`?  I believe `giturlparse` has functionality for parsing git urls too.

Reply:
`giturlparse` please. We already have that in our project `requirements.txt`.